src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -1550,6 +1604,13 @@ static void lb_on_response_received_locked(grpc_exec_ctx *exec_ctx, void *arg,
     if (!glb_policy->seen_initial_response &&
         (response = grpc_grpclb_initial_response_parse(response_slice)) !=
             NULL) {
+      /* Don't use fallback any more. */
```

Comment:
Why isn't `serverlist` parsed from the initial response? Is initial response only for setup work?